src/de/dkfz/b080/co/common/COConfig.groovy

```diff
@@ -12,83 +18,83 @@ class COConfig {
 
     public ExecutionContext context
 
-    public COConfig(ExecutionContext context) {
+    COConfig(ExecutionContext context) {
         this.context = context
     }
 
     // This is used so often, it should maybe be part of ExecutionContext.
-    public RecursiveOverridableMapContainerForConfigurationValues getConfigValues() {
+    RecursiveOverridableMapContainerForConfigurationValues getConfigValues() {
         return context.getConfiguration().getConfigurationValues()
     }
 
-    public setConfig(String flagName, String value, String typeName) {
+    def setConfig(String flagName, String value, String typeName) {
```

Comment:
void setConfig() better expresses, that this is a modifying op.

Reply:
I'll fix it :D. That's actually nothing I changed by myself. The code formatter did it and formerly void was missing... though it should be there for clearness! Groovy can be fun.